docs/resources/domain_record.md

```diff
@@ -0,0 +1,240 @@
+---
+page_title: "Scaleway: scaleway_domain_record"
+description: |-
+  Manages Scaleway Domain records.
+---
+
+# scaleway_domain_record
+
+Creates and manages Scaleway Domain record.  
+For more information, see [the documentation](https://www.scaleway.com/en/docs/scaleway-dns/).
+
+## Examples
+
+### Basic
+
+```hcl
+resource "scaleway_domain_record" "www" {
+  dns_zone = "domain.tld"
+  name     = "www"
+  type     = "A"
+  data     = "1.2.3.4"
+  ttl      = 3600
+}
+
+resource "scaleway_domain_record" "www2" {
+  dns_zone = "domain.tld"
+  name     = "www"
+  type     = "A"
+  data     = "1.2.3.5"
+  ttl      = 3600
+}
+
+resource "scaleway_domain_record" "mx" {
+  dns_zone = "domain.tld"
+  name     = ""
+  type     = "MX"
+  data     = "mx.online.net."
+  ttl      = 3600
+  priority = 10
+}
+
+resource "scaleway_domain_record" "mx2" {
+  dns_zone = "domain.tld"
+  name     = ""
+  type     = "MX"
+  data     = "mx-cache.online.net."
+  ttl      = 3600
+  priority = 20
+}
+```
+
+### With dynamic records
+
+```hcl
```

Comment:
Could you have a section and subsection heading and throw in some explanation about the different types of record? Do not hesitate to have several links pointing back at the scaleway documentation.

Reply:
For the dynamic record, it's explained below, under "## Arguments Reference" > "**Dynamic records:**".
For that "classic"  types (A, AAAA, TXT...) they are not explained in the developer's doc, I don't think this is really relevant here, you don't add a record type you don't already know.